2_process.R

```diff
@@ -24,17 +25,58 @@ p2_targets_list <- list(
     command = save_target_ind_files("2_process/log/wqp_data_ind.csv","p2_wqp_data_subset"),
     format = "file"),
   
+  # Aggregate instantaneous DO data to daily min/mean/maxs
+  tar_target(
+    p2_inst_data_daily,
+    aggregate_data_to_daily(p1_inst_data,p1_daily_data, min_daily_coverage=0.5, output_tz="America/New_York")),
+
+  # Combine 1) daily DO data and 2) instantaneous DO data that has been aggregated to daily 
+  tar_target(
+    p2_daily_combined,
+    bind_rows(p1_daily_data, p2_inst_data_daily)),
+  
   # Create a list of unique site locations containing DO data  
   tar_target(
     p2_site_list_csv,
     create_site_list(p2_filtered_wqp_data_subset,p1_nwis_sites,p1_daily_data,p1_inst_data,
-                       hucs=drb_huc8s,crs_out="NAD83",fileout = "2_process/out/DRB_DO_sitelist.csv"),
-    format = "file"),
-  
+                       hucs=drb_huc8s,crs_out="NAD83")
+  ), 
+
   # Create and save log file containing data availability summary
   tar_target(
     p2_sitelist_summary_csv,
     summarize_site_list(p2_site_list_csv,p1_daily_data,p1_inst_data,fileout = "2_process/log/sitelist_summary.csv"),
-    format = "file")
+    format = "file"),
+
+  tar_target(
```

Comment:
Would you mind adding some one-line comments above each `tar_target()` call to describe the target? Nothing fancy, but I've noticed these commented descriptors help me to quickly navigate the phase files without needing to closely examine each target.

Reply:
Thanks for adding those!